static/js/notifications.js

```diff
@@ -402,6 +402,17 @@ function message_is_notifiable(message) {
         return false;
     }
 
+    // The notification should only be displayed once. To prevent multiple
+    // tabs from opening a notification each for their own, we store the
+    // notification ID in localStorage and check if a notification was
+    // already opened.
+    if (typeof(Storage) !== "undefined") {
```

Comment:
Oops, I didn't read the definition carefully enough.  Should be `localstorage.supported()`.  Basically, within foo.js, `exports` refers to the things in the `foo` module that other modules can access, and writing `exports.bar = ...` means you can access the thing defined there as `foo.bar` from other modules.

Does that make sense?